package.json

```diff
@@ -99,66 +99,66 @@
     "babel-plugin-transform-class-properties": "^6.11.5",
     "babel-preset-es2015": "^6.6.0",
     "babel-preset-react": "^6.11.1",
-    "babel-register": "^6.8.0",
-    "chai": "^3.5.0",
-    "chai-as-promised": "^5.3.0",
+    "babel-register": "^6.26.0",
+    "chai": "^4.1.2",
+    "chai-as-promised": "^7.1.1",
     "chai-fs": "chaijs/chai-fs",
     "coveralls": "^2.11.12",
-    "cross-env": "^2.0.1",
-    "devtron": "^1.0.1",
+    "cross-env": "^5.2.0",
+    "devtron": "^1.4.0",
     "electron-devtools-installer": "^2.2.4",
     "electron-mocha": "^3.0.0",
-    "electron-packager": "^8.0.0",
-    "electron-rebuild": "^1.5.6",
-    "electron-windows-store": "^0.9.3",
+    "electron-packager": "^12.2.0",
+    "electron-rebuild": "^1.8.2",
+    "electron-windows-store": "^2.0.0",
     "enzyme": "^2.4.1",
     "eslint": "^2.10.2",
     "eslint-config-airbnb": "^9.0.1",
     "eslint-plugin-import": "^1.14.0",
     "eslint-plugin-jsx-a11y": "^1.5.5",
     "eslint-plugin-react": "^5.2.2",
-    "fs-extra": "^0.30.0",
-    "glob": "^7.0.6",
+    "fs-extra": "^7.0.0",
+    "glob": "^7.1.3",
     "gpmdp-electron-winstaller": "1.0.0",
     "gulp": "^3.9.1",
-    "gulp-babel": "^6.1.2",
-    "gulp-clean": "^0.3.2",
-    "gulp-concat": "^2.6.0",
-    "gulp-cssmin": "^0.1.7",
-    "gulp-header": "^1.8.2",
-    "gulp-less": "^3.1.0",
-    "gulp-replace": "^0.5.4",
-    "gulp-uglify": "^2.0.0",
-    "istanbul": "^1.1.0-alpha.1",
-    "jimp": "^0.2.27",
-    "material-design-icons-iconfont": "^2.0.5",
-    "materialize-css": "^0.97.6",
+    "gulp-babel": "^7.0.1",
+    "gulp-clean": "^0.4.0",
+    "gulp-concat": "^2.6.1",
+    "gulp-cssmin": "^0.2.0",
+    "gulp-header": "^2.0.5",
+    "gulp-less": "^3.5.0",
+    "gulp-replace": "^1.0.0",
+    "gulp-uglify": "^3.0.0",
+    "istanbul": "^0.4.5",
+    "jimp": "^0.2.28",
+    "material-design-icons-iconfont": "^3.0.3",
+    "materialize-css": "^1.0.0",
     "mocha": "^3.0.0",
     "mocha-appveyor-reporter": "^0.2.1",
     "mocha-testdata": "^1.2.0",
     "nugget": "^2.0.1",
     "plist": "^2.0.1",
-    "pn": "^1.0.0",
-    "png-to-ico": "^2.0.0",
-    "pre-commit": "^1.1.2",
+    "pn": "^1.1.0",
+    "png-to-ico": "^2.0.1",
+    "pre-commit": "^1.2.2",
     "pre-flight": "^1.0.2",
     "rcedit": "^0.7.0",
     "react-addons-test-utils": "^15.3.1",
-    "remap-istanbul": "0.6.4",
-    "run-sequence": "^1.1.5",
-    "sinon": "^1.17.5",
-    "sinon-chai": "^2.8.0",
+    "remap-istanbul": "0.12.0",
+    "run-sequence": "^2.2.1",
+    "sinon": "^6.0.1",
```

Comment:
This bump skips 5 majors and has caused the test failures we are seeing on linux

Reply:
want me to revert it?